web/src/views/SubmissionPortal/harmonizerApi.ts

```diff
@@ -209,9 +209,16 @@ export class HarmonizerApi {
       modalsRoot: document.querySelector('.harmonizer-style-container'),
       fieldSettings: this._getFieldSettings(),
       columnHelpEntries: ['column', 'description', 'guidance', 'examples'],
+      // we use our own custom help sidebar, so turn off DataHarmonizer's built-in one
+      helpSidebar: {
+        enabled: false,
+      },
     });
     this.footer = new Footer(document.querySelector('#harmonizer-footer-root'), this.dh);
     this.dh.useSchema(this.schema, [], templateName);
+    // Let the Handsontable element take up as much space as it wants, it will be visually
+    // constrained by its containing elements
+    this.dh.hot.updateSettings({ height: 'auto', width: 'auto' });
```

Comment:
I think you might want `100%` here instead of `auto`. `auto` will make the height and width dependent on the children and `100%` will make it dependent on it's parent. Disregard if I misunderstood the comment.